diff --git a/dist/development/chunk-KS7C4IRE.mjs b/dist/development/chunk-KS7C4IRE.mjs
index db270c5d80a2e2a122159a6845f548f2db3cad83..2d79cca7800c31a9e693c7be36a70a35c8a2baa6 100644
--- a/dist/development/chunk-KS7C4IRE.mjs
+++ b/dist/development/chunk-KS7C4IRE.mjs
@@ -9721,7 +9721,8 @@ function Links({ nonce, crossOrigin }) {
       rel: "stylesheet",
       href: criticalCss.href,
       nonce,
-      crossOrigin
+      crossOrigin,
+      suppressHydrationWarning: true
     }
   ) : null, keyedLinks.map(
     ({ key, link }) => isPageLinkDescriptor(link) ? /* @__PURE__ */ React8.createElement(
@@ -9738,7 +9739,8 @@ function Links({ nonce, crossOrigin }) {
         key,
         nonce,
         ...link,
-        crossOrigin: link.crossOrigin ?? crossOrigin
+        crossOrigin: link.crossOrigin ?? crossOrigin,
+        suppressHydrationWarning: true
       }
     )
   ));
diff --git a/dist/production/chunk-3WDNQUW5.mjs b/dist/production/chunk-3WDNQUW5.mjs
index 211c18cd3b21bc394d4023c659c91a194e5e7cb4..473a013ec692aa6239100d1e237a966d27017977 100644
--- a/dist/production/chunk-3WDNQUW5.mjs
+++ b/dist/production/chunk-3WDNQUW5.mjs
@@ -9721,7 +9721,8 @@ function Links({ nonce, crossOrigin }) {
       rel: "stylesheet",
       href: criticalCss.href,
       nonce,
-      crossOrigin
+      crossOrigin,
+      suppressHydrationWarning: true
     }
   ) : null, keyedLinks.map(
     ({ key, link }) => isPageLinkDescriptor(link) ? /* @__PURE__ */ React8.createElement(
@@ -9738,7 +9739,8 @@ function Links({ nonce, crossOrigin }) {
         key,
         nonce,
         ...link,
-        crossOrigin: link.crossOrigin ?? crossOrigin
+        crossOrigin: link.crossOrigin ?? crossOrigin,
+        suppressHydrationWarning: true
       }
     )
   ));
